browser/js/kit/kit.controllers.js

```diff
@@ -1,8 +1,23 @@
-app.controller('KitsController', function ($scope, allKits, allKitImages) {
-	
+app.controller('KitsController', function ($scope, allKits, allKitImages, AuthService, $uibModal, $log) {
+	AuthService.getLoggedInUser().then(function(user){
+		$scope.user= user;
+	})
 	$scope.kits = allKits
 	$scope.images = allKitImages
 	$scope.title= 'All Kits'
+	$scope.open = function (size) {
```

Comment:
Does this function check if the user is actually an admin? Or are we just hiding the button on the UI for now? I see we are getting a user on line 2